### ISSUE TITLE
Fix the "import minidom as microdom" stuff in lore

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3619]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3619|
|Type|task|
|Created|2009-01-15 20:16:54Z|
|Branch|https://github.com/twisted/twisted/tree/lore-microdom-imports-3619|

[#3560](https://github.com/twisted/twisted/issues/3560) switched lore from microdom to minidom.  To avoid lots and lots of extra diff, it left lore using the name "microdom" in most places, but made that actually refer to minidom.

Change all of the names used in Lore to minidom to make the code less confusing.

<details><summary>Searchable metadata</summary>

```
trac-id__3619 3619
type__task task
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_lore_microdom_imports_3619 branches/lore-microdom-imports-3619
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__easy easy
time__1232050614000000 1232050614000000
changetime__1242160079000000 1242160079000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [26745]) Branching to 'lore-microdom-imports-3619'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun removed owner|
|-|-|

I actually changed all the names to `dom` rather than `minidom`.  Maybe this will afford us some more flexibility in changing to yet another DOM implementation in the future!  Or maybe it doesn't matter.

Anyway I changed all the names and the [tests all still pass](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/lore-microdom-imports-3619).

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @exarkun|
|-|-|

`dom` is still used as a parameter name a bunch of times in `twisted.lore.lint`, once in `twisted.lore.slides`, and as a variable in both of those modules as well as `twisted.lore.latex`.

But if the tests still pass once you've renamed those, go ahead and land it.  Alternately, rename the import to something else, like 'minidom' as originally planned.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

(In [26928]) Rename this variable from dom to doc to avoid name collision with global

refs [#3619](https://github.com/twisted/twisted/issues/3619)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [26929]) Merge lore-microdom-imports-3619 

Author: exarkun
Reviewer: glyph
Fixes: [#3619](https://github.com/twisted/twisted/issues/3619)

Change the `minidom as microdom` imports in Twisted Lore to be
`minidom as dom` instead, and adjust all affected code.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
